src/screens/ConstructionSiteDetailScreen.js

```diff
@@ -0,0 +1,162 @@
+  const formattedDates = formatDates(startDate, endDate);
+
+  return (
+    <SafeAreaViewFlex>
+      <ScrollView keyboardShouldPersistTaps="handled">
+        <WrapperWithOrientation>
+          {!!imageUri && <Image source={{ uri: imageUri }} />}
+          <TitleContainer>
+            <Title>{extendedTitle}</Title>
+          </TitleContainer>
+          <Wrapper>
+            <View style={styles.informationContainer}>
+              <BoldText>{endDate ? 'Gesamtzeitraum: ' : 'Datum: '}</BoldText>
+              <RegularText>{formattedDates}</RegularText>
+            </View>
+            {!!locationDescription && (
+              <View style={styles.informationContainer}>
+                <BoldText>Standort: </BoldText>
+                <RegularText>{locationDescription}</RegularText>
+              </View>
+            )}
+            {!!direction && (
+              <View style={styles.informationContainer}>
+                <BoldText>Richtung: </BoldText>
+                <RegularText>{direction}</RegularText>
+              </View>
+            )}
+            {!!cause && (
+              <View style={styles.informationContainer}>
+                <BoldText>Ursache: </BoldText>
+                <RegularText>{cause}</RegularText>
+              </View>
+            )}
+            {!!description && (
+              <View style={styles.verticalPadding}>
+                <BoldText>Weitere Informationen: </BoldText>
+                <RegularText>{description}</RegularText>
+              </View>
+            )}
+            {!!restrictions?.length && (
+              <View style={styles.verticalPadding}>
+                <BoldText>Aktuelle Einschränkungen: </BoldText>
+                {restrictions.map((restriction) => (
+                  <RegularText key={'restriction-' + index}>- {restriction}</RegularText>
+                ))}
+              </View>
+            )}
+          </Wrapper>
+          {!!location && (
+            <WebViewMap
+              locations={[
+                {
+                  icon: locationIcon(colors.primary),
+                  iconAnchor: locationIconAnchor,
+                  position: { lat: location.lat, lng: location.lon }
+                }
+              ]}
+            />
+          )}
+        </WrapperWithOrientation>
+      </ScrollView>
+    </SafeAreaViewFlex>
+  );
+};
+
+ConstructionSiteDetailScreen.navigationOptions = ({ navigation }) => {
+  return {
+    headerLeft: (
+      <View>
+        <TouchableOpacity
+          onPress={() => navigation.goBack()}
+          accessibilityLabel="Zurück Taste"
+          accessibilityHint="Navigieren zurück zur vorherigen Seite"
+        >
+          <Icon xml={arrowLeft(colors.lightestText)} style={styles.icon} />
+        </TouchableOpacity>
+      </View>
+    )
+  };
+};
+
+const styles = StyleSheet.create({
+  informationContainer: {
+    flexDirection: 'row',
+    flexWrap: 'wrap'
```

Comment:
we could think about creating a styled-components wrapper for this container.
we already have a `WrapperWrap` with the following styles, which maybe works?

```
flex-direction: row;
flex-wrap: wrap;
justify-content: space-between;
width: 100%;
```

otherwise we can adjust/add things.

Reply:
i personally find it awkward having the information be pushed to the right edge of the screen/view due to it being space between.

besides that it should work.

i could add a 'spaceBetween' prop to WrapperWrap and use that prop everywhere but here.

What are your thoughts?

Reply:
yes sure, should not use `space-between` here, if this breaks the layout. that was not my intention.

Reply:
keep it with the `informationContainer` as is.
we do not want to update every occurrence of `WrapperWrap` now with figuring out a best way with props.
and `WrapperRow` is missing the `wrap` part, so there is no wrapper component existing, that can be used here, right?!

Reply:
after i had now changed it i saw this comment, so you can decide whether or not to keep the last commit :-P

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/157/commits/505a20527fa5051b0ce55ed3db9b7259464e5654

ok. than we keep your work for sure. these four places are fine i would say.